datasets/wegenbestand/dataset.json

```diff
@@ -17,10 +17,10 @@
   "dateCreated": "2022-02-09T00:00:00+01:00",
   "license": "Creative Commons, Naamsvermelding",
   "hasBeginning": "2022-02-09T00:00:00+01:00",
-  "accrualPeriodicity": "dagelijks",
-  "creator": "bronhouder onbekend",
-  "publisher": "Datateam Generiek",
-  "description": "Aangewezen wegen voor vrachtverkeer met gevaarlijkestoffen of valt onder de categorie zwaar verkeer.",
+  "accrualPeriodicity": "per kwartaal",
+  "creator": "@Verkeersonderzoek",
```

Comment:
According to the ticket, this should be
`"creator": "verkeersonderzoek@amsterdam.nl",`